composer.json

```diff
@@ -30,7 +30,7 @@
     ],
     "license": "GPL-2.0-or-later",
     "require": {
-        "php": ">=7.4.1",
+        "php": "^7.4",
```

Comment:
As you can see from the failing tests, that is something totally different. You now marked it only compatible with PHP >=7.4.0 but < 8. It won't be installable with PHP 8 and higher then. 

Additionally, as pointed out in the other PR already, I won't merge a change that reduces the PHP requirement.